lib/Driver/Driver.cpp

```diff
@@ -669,6 +669,15 @@ Driver::buildCompilation(const ToolChain &TC,
                      A->getAsString(*ArgList), A->getValue());
     }
   }
+  if (const Arg *A = ArgList->getLastArg(options::OPT_driver_batch_size)) {
```

Comment:
I would have factored this out into a static function, in order to keep a large routine (buildCompilation) from getting larger.

Reply:
fixed. also localized the flag and two others to the function, as they were previously (pointlessly) members of Driver only to be passed off to members of Compilation.

Reply:
Great!